packages/yoroi-extension/app/api/ada/lib/state-fetch/remoteFetcher.js

```diff
@@ -427,4 +429,25 @@ export class RemoteFetcher implements IFetcher {
         throw new GetCatalystRoundInfoApiError();
       });
   }
+
+  getMultiAssetMintMetadata: MultiAssetMintMetadataRequest
+    => Promise<MultiAssetMintMetadataResponse> = async (body) => {
+      const { BackendService } = body.network.Backend;
+      if (BackendService == null) throw new Error(`${nameof(this.getCatalystRoundInfo)} missing backend url`);
```

Comment:
Please update the name of the function from `this.getCatalystRoundInfo` to `this.getMultiAssetMintMetadata`

Reply:
Done!